lib/migration.js

```diff
@@ -701,6 +701,12 @@ function mixinMigration(MySQL, mysql) {
     var p = this.getModelDefinition(model).properties[prop];
     var line = this.columnDataType(model, prop) + ' ' +
       (this.isNullable(p) ? 'NULL' : 'NOT NULL');
+    if (typeof p.mysql !== 'undefined' && p.mysql.default && typeof p.mysql.default === 'string') {
+      line = line + ' DEFAULT ' +
+            (p.mysql.default.toUpperCase() === 'CURRENT_TIMESTAMP' || p.mysql.default.toLowerCase() === 'now' ?
+              'CURRENT_TIMESTAMP' : '"' + p.mysql.default + '"');
```

Comment:
It also says these types aren't support:
>BLOB, TEXT, and JSON columns cannot be assigned a default value.

I can add logic to prevent these types from attempting to set a default.